.github/workflows/test.yml

```diff
@@ -94,9 +93,16 @@ jobs:
       run: |
         chmod +x bin/wego
         ls -la bin
+    - name: Download dependecies
+      run: |
+        make dependencies
     - name: Install Ginkgo binary
```

Comment:
@luizbafilho please add this block here

    - name: Set up ssh agent
      uses: webfactory/ssh-agent@v0.5.2
      with:
        ssh-private-key: ${{ secrets.WKP_EXAMPLE_ORG_SSH_KEY }}
    - name: Configure git settings
      run: |
        export WEGO_BIN_PATH=$(pwd)/bin/wego-${{matrix.os}}
        git config --global init.defaultBranch main
        git config --global user.email "test-user@weave.works"
        git config --global user.name "Testy McTestFace"
        git config --global url.ssh://git@github.com/.insteadOf https://github.com/

Reply:
thanks